README.md

```diff
@@ -0,0 +1,11 @@
+# Okular (org.kde.okular)
+
+## Permissions rationale
+
+This application requests the following additional permissions:
+
+- Access to the host filesystem: `--filesystem=host`.
+  This is required for drag & drop support and to enable openinf files from
```

Comment:
typo `openinf` -> `opening`. 

Also `--filesystem=host` doesn't open access to `/tmp`. IIRC recent firefox doesn't use `/tmp` anymore. I would just end this descripton on _This is required for drag & drop support_.

Reply:
Thanks!